src/components/ForestTypeModal/ForestTypeComparison/bl/ForestTypeTab.js

```diff
@@ -0,0 +1,318 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Table } from 'semantic-ui-react';
+import { info } from '@geops/tree-lib';
+import useIsMobile from '../../../../hooks/useIsMobile';
+
+import ComparisonCell from '../ComparisonCell';
+import ForestTypeLink from '../../ForestTypeLink';
+import ForestTypeLinksList from '../../ForestTypeLinksList';
+import Site from '../../ForestTypeDescription/bl/Site';
+import {
+  setForestTypeDescription,
+  setForestTypeModal,
+} from '../../../../store/actions';
+import {
+  treeTypeMapping,
+  vegetationMapping,
+} from '../../ForestTypeDescription/bl/utils';
+import SoilIcon from '../../../../icons/SoilIcon';
+import comparisonStyles from '../ForestTypeComparison.module.css';
+
+const getTreeTypes = (treeTypes, category) =>
+  treeTypeMapping.reduce(
+    (finalTreeTypes, treeType, index) =>
+      treeTypes[index] === category
+        ? `${finalTreeTypes}${`${finalTreeTypes ? ', ' : ''}${treeType}`}`
+        : finalTreeTypes,
+    '',
+  );
+
+const HeaderCell = ({ ...props }) => {
```

Comment:
Can we extract some of this into generic components or util functions?

Reply:
Done, I refactored a lot of the Table.Cell and Table.Row components